libminifi/include/utils/TimeUtil.h

```diff
@@ -42,6 +42,10 @@
 
 #include "date/date.h"
 
+#ifdef WIN32
+#include "date/tz.h"
+#endif
```

Comment:
Why is this windows-only? The `dateSetInstall` function declaration doesn't use anything from this header, so if that's the culprit, then this include should be moved to the cpp file.

Reply:
It was moved from the TestUtils and it only contained `dateSetInstall` that was WIN32 specific, so it was probably due to this. I moved it to the cpp file in 82084145ce5f0dd020144aea53c0c791bb69d3aa